src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,8 +28,11 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here
   private ElevatorSubsystem elevatorSubsystem;
+  private RampSubsystem rampSubsystem = new RampSubsystem(new RampMechanism(new RampIOSim()));
```

Comment:
Feature flags need to be added.
Also this should be instantiated based on if this is sim or robot (see other subsystems for examples)